packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -71,26 +73,39 @@ class GoogleSignInPlugin extends GoogleSignInPlatform {
     SignInOption signInOption = SignInOption.standard,
     String? hostedDomain,
     String? clientId,
-  }) async {
-    final String? appClientId = clientId ?? _autoDetectedClientId;
+  }) {
+    return initWithParams(SignInInitParameters(
+      scopes: scopes,
+      signInOption: signInOption,
+      hostedDomain: hostedDomain,
+      clientId: clientId,
+    ));
+  }
+
+  @override
+  Future<void> initWithParams(SignInInitParameters params) async {
+    final String? appClientId = params.clientId ?? _autoDetectedClientId;
     assert(
         appClientId != null,
         'ClientID not set. Either set it on a '
         '<meta name="google-signin-client_id" content="CLIENT_ID" /> tag,'
-        ' or pass clientId when calling init()');
+        ' or pass clientId when calling initialize()');
```

Comment:
Did the `init` method get renamed to `initialize`?

Reply:
Good catch. I made this change mistakenly at an earlier point when `initWithParams` was called `initialize`. I think referencing a method of the platform interface is not very helpful to the user, though. I slightly changed the wording to "pass clientId when initializing GoogleSignIn".